lib/core/src/lib/dialogs/dialog/dialog-data.interface.ts

```diff
@@ -35,6 +35,7 @@ export interface DialogData {
     descriptionTemplate?: TemplateRef<any>;
     headerIcon?: string;
     additionalActionButtons?: AdditionalDialogActionButton[];
+    componentData?: any;
```

Comment:
I understand that you want to use 'any' because Dialog can use any data. But can we maybe use some generic type instead? I mean something like: 
export interface DialogData<T> {
  componentData?: T;
}

where T is generic type. It will be a bit safer.

Reply:
discussed on slack, so closing comment